Tutorial.md

```diff
@@ -11,477 +10,251 @@ Gist is a paste. A snippet of text. Not unrelated to [GitHub Gists][].  To set s
-
-#### Response
-
-    + Response 200
-
-In API Blueprint an action **should** always include at least one response representing the HTTP response message sent back in response to a HTTP request. The response should always bear a [status code][] and possibly an additional [payload][]. Here we have defined the most common response "200" indicating the request has succeeded.
-
-#### Response Payload
-
-    + Response 200 (application/hal+json)
-        + Headers
-
-                Link: <http:/api.gistfox.com/>;rel="self",<http:/api.gistfox.com/gists>;rel="gists"
+                ]
+            }
+        ]
+```
 
-        + Body
+> **Note:** Specifying the media type after the response status code generates a `Content-Type` HTTP header. You do not have to explicitly specify the `Content-Type` header.
 
-                {
-                    "_links": {
-                        "self": { "href": "/" },
-                        "gists": { "href": "/gists?{since}", "templated": true }
-                    }
-                }
+The polls resource has a second action which allows you to create a new question. This action includes a description showing the structure you would send to the server to perform this action.
 
-A response usually bears some payload sent to our client. Ideally it details the resource in question. In this case our response representation is of the [`application/hal+json`][] media-type. Also in addition to an HTTP response message-body this response defines an HTTP response [message-headers][] in its payload.
+```markdown
+### Create a New Question [POST]
 
-> **Note:** API Blueprint is **indentation sensitive**. See the [Note on Indentation][] for details.
->
-> **Note:** Specifying the media type in parenthesis after the response status codes generates implicit `Content-Type` HTTP header. Therefore you do not have to explicitly specify the `Content-Type` header.
->
-> **Note:** If your response does not need to define any additional headers (but `Content-Type`) you can skip the `Headers` section completely and write the `Body` section like so:
->
->     + Response 200 (application/hal+json)
->
->         {
->             "_links": {
->                 "self": { "href": "/" },
->                 "gists": { "href": "/gists?{since}", "templated": true }
->             }
->         }
+You may create your own question using this action. It takes a JSON object containing a question and a collection of answers in the form of choices.
 
-## Complex Resource
-With the entry point of our API defined we can move forward. Since our API revolves around a Gist let's define its representation and an action to retrieve it:
++ question (string) - The question
++ choices (array[string]) - A collection of choices.
+```
 
-    # Group Gist
-    Gist-related resources of *Gist Fox API*.
+> **Note:** Lists are created by preceding list items with either a `+`, `*` or `-`.
```

Comment:
I would expect a bigger "list" introduction (one sentence will do). Possibly on its first use – when you are explaining action's response?